templates/password.json

```diff
@@ -4,13 +4,27 @@
     "main": {
       "type": "newsletter",
       "settings": {
-        "heading": "Opening soon",
-        "subheading": "<p>Be the first to know when we launch.</p>",
         "full_width": false
-      }
+      },
+      "blocks": {
+        "heading": {
+          "type": "heading",
+          "settings": {
+            "heading": "Opening soon"
+          }
+        },
+        "subheading": {
+          "type": "paragraph",
+          "settings": {
+            "paragraph": "<p>Be the first to know when we launch.</p>"
```

Comment:
Can we not instead use the [default value](https://github.com/Shopify/dawn/blob/main/sections/newsletter.liquid#L124) that's already defined?

Reply:
I don't think so as this is more specific to the password page. Otherwise the default text would be: `Be the first to know about new collections and exclusive offers.`. Which isn't too bad but not as on point I think. 

I can chat it over with a designer if needed 🤔

Reply:
I think `Be the first to know when we launch.` was suggested by @gretchen-willenberg I'm ok with using the other sentence if we have to, but I think this one's better for the password  page.